README.md

```diff
@@ -16,50 +16,61 @@
+perform physics studies from _developing_ it to fix/improve/enable other studies.
+In both cases, we use containers to share a fixed software environment, so everyone
+will need a method for running these containers.
 
 - [Install the docker engine](https://docs.docker.com/engine/install/)
-- (on Linux systems) [Manage docker as non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
-- (on MacOS systems) Make sure `git lfs` is installed. (Test: `git lfs` prints out a help message instead of an error about `lfs` not being found.)
-  - The default installation of `git` that is included with Apple's developer tools does not include `git lfs` which is required by acts to download and unpack one of its own submodules. [GitHub has a nice tutorial](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage?platform=mac) on how to install `git lfs` on MacOS.
-  - This is only an issue observed on MacOS systems. Linux repositories (used in WSL and Linux systems) include `git lfs` within their installed versions of `git`.
-- Clone the repo: `git clone --recursive git@github.com:LDMX-Software/ldmx-sw.git`
-  - **Note**: You need to [setup an SSH-key with your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh) on the computer you are using.
-- Setup the environment (in bash): `source ldmx-sw/scripts/ldmx-env.sh`
-  - **Note**: If you are working with ldmx-sw at SLAC's SDF, you will need to set the `TMPDIR` environment variable so that program running the container has more than ~5GB of space to write intermediate files. The default temporary space (`/tmp`) is often full of other files already. A decent replacement is `TMPDIR=${SCRATCH}` which gives the program plenty of room for the files it needs to manipulate.
-- Configure and compile: `ldmx compile`
-- Now you can run any processor in _ldmx-sw_ through `ldmx fire myconfig.py`
-- If you are developing and need to recompile and run `ldmx fire`, you can use `ldmx recompFire myconfig.py`
+  - Only necessary on personal computers. Shared computing clusters should have `apptainer` installed.
+  - (on Linux personal computers) [Manage docker as non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
+- [Install `denv`](https://tomeichlersmith.github.io/denv/getting_started.html#installation)
 
-## Documentation 
-The full documentation for **ldmx-sw** is available on [github pages](https://ldmx-software.github.io/).
-A brief description of common commands is given below.
+### Users
+In order to use ldmx-sw, no more dependencies are required!
+Simply choose the version of ldmx-sw you wish to use with your project.
+```
+mkdir my-project
+cd my-project
+denv init ldmx/pro:v4.0.1 # or some other ldmx-sw version
+```
+And then you can run ldmx-sw with a configuration script of your choice.
+```
+denv fire my-config.py
```

Comment:
it would be nice if this also could become `ldmx fire`

Reply:
Since with `just`, I understand we will use `ldmx fire` thru an alias, so if this stays like this, that means running fire will be with two different options depending if we are using the image or the cloned repo, right?

Reply:
OK this might be too much to ask :D but if I understand you plan to have a fork of `denv` under ldmx-sw, can we have it with the name `ldmx` instead? Like that the transition is very smooth, people have been calling `ldmx` in the past to get the environment, and they'd do the same from now on too, it's just the under the hood that's changed.

Reply:
It's not too much to ask, its just a stylistic choice. The main reasons I'm resistant to propagating an alias/wrapper/symlink of `denv` to `ldmx` are

1. While it maintains some backwards compatibility, there is little self-discovery. i.e. Users of `ldmx` (which simply redirects to `denv`) would not be directed towards denv's documentation and therefore potentially miss out on helpful information.
2. The implication is restrictive. `ldmx` implies that it is special to LDMX and ldmx-sw but `denv` does not. It (correctly in my opinion) implies that other projects can use the same software interaction workflow in order to facilitate easier development. I particularly am fond of this reason - I use `denv` to enable using identical and modern Python versions on the UMN cluster, SLAC cluster, and my personal computers which has made it so much easier to share my analysis and plotting code. This is something that I would forsee being useful for everyone - containers can be helpful for portability and reproducibility not just in ldmx-sw but in all projects that write any kind of code (even LaTeX lmao).

These reasons being stated, I could imagine the LDMX-Software fork of `denv` including a symlink to `denv` named `ldmx` within the `install` script so that users could do `ldmx ...` in place of `denv ...` if they desire to out of habit.

Reply:
Yes, I understand `denv` is bigger than `ldmx`, I think supporting both would be nice, indeed at this point it's more about my habits than anything else

Reply:
That being said I'm more likely to use the `just` approach anyway since I seem to be forever developing something. Given that we have an alias there for `ldmx` this might lead to a conflict? So maybe the symlink should be `ldmxenv` for `denv` and we keep the alias of `ldmx` for `just`?

Reply:
There would not be a conflict in most[^1] shells. The alias would overwrite the symlink (which acts as a program). The symlink would be accessible with `command ldmx` (`command` avoids shell functions/aliases).

[^1]: I believe its a POSIX sh requirement that aliases/functions can have the same names as programs.